src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+const calendar = document.querySelector('.calendar');
+
+const daysInMonth = 31;
+
+function generateCalendar(daysInMonth) {
+
+  // Add divs for each day in the month
+  for (let i = 1; i <= daysInMonth; i++) {
+    const day = document.createElement('div');
+    day.classList.add('calendar__day');
+    day.setAttribute('data-day', i);
+    calendar.appendChild(day);
+  }
+}
+
+generateCalendar(daysInMonth);
```

Comment:
You are calling the function `generateCalendar` immediately after defining it. This is not a good practice because it makes the code less flexible and harder to test. Consider moving this call to a separate script tag at the bottom of your HTML file or wrapping it in an `DOMContentLoaded` event listener.